guides/v2.3/install-gde/system-requirements-tech.md

```diff
@@ -77,7 +77,7 @@ For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde
 
 * A valid [security certificate](https://glossary.magento.com/security-certificate) is required for HTTPS.
 * Self-signed SSL certificates are not supported.
-* Transport Layer Security (TLS) requirement - PayPal and `repo.magento.com` both require TLS 1.1 or later:
+* Transport Layer Security (TLS) requirement - PayPal and `repo.magento.com` both require TLS 1.2 or later:
```

Comment:
Hi @jonashrem. Thank you for the modification, it completely makes sense.
This update is fair for both Magento 2.2.x and Magento 2.3.x. So, I would kindly ask you to provide the change for 2.2 as well. 

Thank you!

Reply:
sure